htdocs/install/mysql/tables/llx_product_fournisseur_price.key.sql

```diff
@@ -25,7 +25,7 @@ ALTER TABLE llx_product_fournisseur_price ADD CONSTRAINT fk_product_fournisseur_
 
 -- Added to remove table llx_product_fournisseur
 
-ALTER TABLE llx_product_fournisseur_price ADD UNIQUE INDEX uk_product_fournisseur_price_ref (ref_fourn, fk_soc, quantity, entity);
+ALTER TABLE llx_product_fournisseur_price ADD UNIQUE INDEX uk_product_fournisseur_price_ref (ref_fourn, fk_soc, quantity, entity, fk_product);
```

Comment:
This looks strange that the same supplier send 2 different products with same supplier ref ?

Reply:
Yes, this allow to have different products (for example : old and new internal products ) for the same supplier reference